### PR TITLE
Always exclude node modules

### PR DESCRIPTION
**Pull request by hoanhtien**
* Remove 'exclude node_modules' checkbox. Only the checkbox. I am not removing the data model behind the screen e.g. ExcludeNodeModulesProperty in case we need it back in the future.
* Exclude node_modules subfolders.
* Exclude bower_components folder/subfolders.

This fixes https://github.com/Microsoft/nodejstools/issues/430.

### CODE REVIEW

File: Nodejs/Product/Nodejs/Project/ImportWizard/FileSourcePage.xaml [hunk 1]
```diff
@@ -52,12 +52,5 @@
                 </ListView.ItemTemplate>
             </ListView>
         </wpf:LabelledControl>
-        
-        <wpf:LabelledControl Title="Select any other options for your project."
-                           Grid.Row="1">
-            <StackPanel Orientation="Vertical">
-                <CheckBox IsChecked="{Binding ExcludeNodeModules}">Exclude node__modules</CheckBox>
```
Comment (mousetraps) [inline Nodejs/Product/Nodejs/Project/ImportWizard/FileSourcePage.xaml]: You should remove the ExcludeNodeModules binding as well if it's no longer being used.

File: Nodejs/Product/Nodejs/Project/ImportWizard/ImportSettings.cs [hunk 2]
```diff
@@ -244,6 +244,16 @@ internal class ImportSettings : DependencyObject {
             // We won't reject the folder.
             Debug.Assert(!Path.IsPathRooted(dirName));
             return !dirName.Split(new char[] { '/', '\\' }).Any(name => name.StartsWith("."));
+        }
+
+        /// <summary>
+        /// Checks whether a relative and double-backslashed seperated path contains a folder name.
+        /// </summary>
+        private static bool ContainsNodeModulesOrBowerComponentsFolder(string path)
+        {
```
Comment (mousetraps) [inline Nodejs/Product/Nodejs/Project/ImportWizard/ImportSettings.cs]: braces on same line here and everywhere else

File: Nodejs/Product/Nodejs/Project/ImportWizard/FileSourcePage.xaml (hunk 1 above)
Comment (mousetraps) [inline Nodejs/Product/Nodejs/Project/ImportWizard/FileSourcePage.xaml]: Also verify that the ImportWizardTests continue to work.

**Comment by hoanhtien** [comment]
Addressed feedback.

**Pull request closed without merging**